main.go

```diff
@@ -0,0 +1,197 @@
+
+	// Provide the name of the database and collection you want to use.
+	// If they don't already exist, the driver and Atlas will create them
+	// automatically when you first write data.
+	var dbName = "myDatabase"
+	var collectionName = "recipes"
+	collection := client.Database(dbName).Collection(collectionName)
+
+	/*      *** INSERT DOCUMENTS ***
+	 *
+	 * You can insert individual documents using collection.Insert().
+	 * In this example, we're going to create 4 documents and then
+	 * insert them all in one call with InsertMany().
+	 */
+
+	eloteIngredients := make([]string, 0)
+	eloteIngredients = append(eloteIngredients, "corn")
+	eloteIngredients = append(eloteIngredients, "mayonnaise")
+	eloteIngredients = append(eloteIngredients, "cotija cheese")
+	eloteIngredients = append(eloteIngredients, "sour cream")
+	eloteIngredients = append(eloteIngredients, "lime")
+
+	eloteRecipe := Recipe{"elote", eloteIngredients, 35}
+
+	locoMocoIngredients := make([]string, 0)
+	locoMocoIngredients = append(locoMocoIngredients, "ground beef")
+	locoMocoIngredients = append(locoMocoIngredients, "butter")
+	locoMocoIngredients = append(locoMocoIngredients, "onion")
+	locoMocoIngredients = append(locoMocoIngredients, "egg")
+	locoMocoIngredients = append(locoMocoIngredients, "bread bun")
+	locoMocoIngredients = append(locoMocoIngredients, "mushrooms")
+
+	locoMocoRecipe := Recipe{"loco moco", locoMocoIngredients, 54}
+
+	patatasBravasIngredients := make([]string, 0)
+	patatasBravasIngredients = append(patatasBravasIngredients, "potato")
+	patatasBravasIngredients = append(patatasBravasIngredients, "tomato")
+	patatasBravasIngredients = append(patatasBravasIngredients, "olive oil")
+	patatasBravasIngredients = append(patatasBravasIngredients, "onion")
+	patatasBravasIngredients = append(patatasBravasIngredients, "garlic")
+	patatasBravasIngredients = append(patatasBravasIngredients, "paprika")
+
+	patatasBravasRecipe := Recipe{"patas bravas", patatasBravasIngredients, 80}
+
+	friedRiceIngredients := make([]string, 0)
+	friedRiceIngredients = append(friedRiceIngredients, "rice")
+	friedRiceIngredients = append(friedRiceIngredients, "soy sauce")
+	friedRiceIngredients = append(friedRiceIngredients, "egg")
+	friedRiceIngredients = append(friedRiceIngredients, "onion")
+	friedRiceIngredients = append(friedRiceIngredients, "pea")
+	friedRiceIngredients = append(friedRiceIngredients, "carrot")
+	friedRiceIngredients = append(friedRiceIngredients, "sesame oil")
+
+	friedRiceRecipe := Recipe{"fried rice", friedRiceIngredients, 40}
+
+	// Create an interface of all the created recipes
+	recipes := []interface{}{eloteRecipe, locoMocoRecipe, patatasBravasRecipe, friedRiceRecipe}
+	insertManyResult, err := collection.InsertMany(context.TODO(), recipes)
+	if err != nil {
+		log.Fatal("Something went wrong trying to insert the new documents: ", err)
+	}
+	fmt.Println(insertManyResult.InsertedIDs, " documents successfully inserted.")
```

Comment:
Does `InsertedIDs` print the ids or number of documents? I think it's the former which doesn't quite match with the text.

Reply:
It prints the ids, I've switched the code to "fmt.Println("the following documents were inserted:", insertManyResult.InsertedIDs)"  now